Jenkinsfile_k8s

```diff
@@ -0,0 +1,62 @@
+def terraformTemplate = """
+apiVersion: v1
+kind: Pod
+spec:
+  containers:
+  - name: terraform
+    image: hashicorp/terraform:0.13.5
+    command:
+    - sleep
+    args:
+    - infinity
+    volumeMounts:
+      - name: workspace-volume
+        mountPath: /home/jenkins/agent
+    workingDir: "/home/jenkins/agent"
+  volumes:
```

Comment:
Newbie question here: what is the reason of using a PVC instead of a local volume?